schema_enforcer/instances/file.py

```diff
@@ -97,7 +97,24 @@ def top_level_properties(self):
         """
         if not self._top_level_properties:
             content = self._get_content()
-            self._top_level_properties = set(content.keys())
+            # TODO: Investigate and see if we should be checking this on initialization if the file doesn't exists or is empty.
+            if not content:
```

Comment:
Should this be something like `if content is None`? Or is it intentional that if `content` is an empty dict, empty str, or empty list then `_top_level_properties` is unchanged?